.github/workflows/layered-build.yaml

```diff
@@ -20,4 +20,29 @@ jobs:
                   path: element-web/webapp
                   # We'll only use this in a triggered job, then we're done with it
                   retention-days: 1
-
+    cypress:
```

Comment:
ah yes, unfortunately I think it can only run after a job in the same workflow file (apart from the workflow_run job but that runs with too much permission). Meant to rename the workflow file, and have now done so.